source/openapi-admin-v3.yaml

```
@@ -351,6 +351,109 @@ paths:
     parameters:
       - "$ref": "#/components/parameters/GroupId"
       - "$ref": "#/components/parameters/AppId"
+  
+  "/groups/{groupId}/apps/{appId}/event_subscriptions":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+      - name: type
+        description: |-
+          The type of event subscription to return.
+        in: query
+        required: false
+        schema:
+          type: string
+          enum:
+            - DATABASE
+            - AUTHENTICATION
+            - SCHEDULED
+            - SYNCTRANSLATOR
+          x-enumDescriptions:
+            DATABASE: Return database triggers
+            AUTHENTICATION: Return authentication triggers
+            SCHEDULED: Return scheduled triggers
```

```suggestion
            DATABASE: Return Database Triggers
            AUTHENTICATION: Return Authentication Triggers
            SCHEDULED: Return Scheduled Triggers
```

product names Title Case!